nixos/modules/services/misc/gitolite.nix

```diff
@@ -49,6 +49,30 @@ in
         '';
       };
 
+      manageGitoliteRc = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          Make the <literal>~/.gitolite.rc</literal> configuration file
+          managed by NixOS.
+        '';
+      };
+
+      extraGitoliteRc = mkOption {
+        type = types.lines;
+        default = "";
+        example = literalExample ''
+          push( @{$RC{ENABLE}}, 'Kindergarten' );
```

Comment:
Would you mind adding this line to the example?

```
$RC{UMASK} = 0027;
```

(To help out complete Perl dummies like me.)

Reply:
Done. Also added some more helpful examples.